tac/project-updates/2024/2024-Q4-Hyperledger-Identus.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+---
+Created by Javier Ribó with input from other maintainers
+
+
+# Project Health
+
+The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.
+The project can now be found on the [LFX dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true) which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.
+
+
+# Questions/Issues for the TOC
+
+N/A
+
+# Releases
+
+  * Identus V2.13
+    * Cloud Agent v1.39.0
+    * PRISM Node v2.5.0
+    * Mediator v1.0.0
+    * Apollo Crypto Library v1.4.3
+    * Edge Agent SDK Swift v6.1.0
+    * Edge Agent SDK TypeScript v6.1.0
+    * Edge Agent SDK KMP v4.0.0
+    * Documentation Portal v1.110
+
+  * In the coming days the following releases are planned:
+    * Prism Node 3.0.0
+    * Mediator 1.0.0
+    * Cloud Agent 1.40.0
+
+
+# Overall Activity in the Past Quarter
+
+During the previous quarter the team has been focusing on the following:
+1. Connectionless Issuance and Verification of VC via DIDCommV2
+2. DID Prism AnonCreds method to resolve the resources from the DID Prism URL (for the schema and credential definition)
+3. Crypto Agility of the DID Prism Method (added Ed25519 cryptography suite)
+4. Transitioning the project from the IOHK to the Hyperledger organization
+5. OID4VCI specification implementation in the Cloud Agent and SDKs (for JWT VC only)
+6. Major refactoring: 
+    a. improved error handling in the Cloud Agent, the Mediator, and SDKs, 
+    b. added Problem Report protocol to other protocols,
+    c. Background jobs are replaced with the message queue (in-memory or Kafka)
+7. SD-JWT-VC implementation in the Cloud Agent and SDKs
+8. Status List revocation implementation in JWT VC
+9. Improved documentation with the Quick Start Guide and Tutorials
+
+
+
+# Current Plans
+
+As the IOG team that acted as maintainers of Identus has scaled down, the objective during this quarter will be to encourage more participation from the community in both development, governance and community engagement
```

Comment:
I think more background on this would be appreciated, as this feels like a "dump and run", given the short period between when Identus was proposed by IOG to be incubated in Hyperledger and the scaling down.

Reply:
Hello @jimthematrix , IOG has decided not to chase commercial objectives with ATALA/Identus and in consequence it has scaled down the team leaving a team of 9 engineers to maintain and keep evolving Identus.
There isn't any additional background to add beyond that.

Reply:
Let's try to rephrase the sentence to better reflect with Daniel mention

Reply:
```suggestion
As IOG shifts its focus away from commercial objectives for ATALA/Identus, the team dedicated to maintaining the project has been reduced. Our goal for this quarter is to foster increased community involvement in development, governance, and engagement. To ensure the continued development and growth of Identus.
```

Or sounds a bit repetitive with the next sentence?

Reply:
thanks for the clarification on ATALA vs. Identus. the new proposed wording helps quite a bit